x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/63_enrich_int_range.yml

```diff
@@ -192,8 +192,17 @@ teardown:
 
 ---
 "Invalid age as double":
+  - requires:
+        test_runner_features: [ capabilities ]
+        capabilities:
+          - method: POST
+            path: /_query
+            parameters: [ ]
+            capabilities: [ esql_allow_partial_results ]
```

Comment:
Not sure I understand - if we have disabled compat test with this in gradle, why do we need this check?

Reply:
There are two different things. The rest-compat tests are from the source code of a previous branch. We need to keep this skip to avoid running this test from the current branch with clusters that don't support partial results.